codespell_lib/data/dictionary.txt

```diff
@@ -21848,6 +21855,8 @@ reviewsectio->reviewsection
 revisisions->revisions
 revison->revision
 revisons->revisions
+reviwer->reviewer
```

Comment:
Looking at my previous review in #1479 (I'm not quite sure why that got closed), here is another oustanding comment:
reviwed->reviewed and reviwing->reviewing

Reply:
I closed it by mistake :-/ Thanks for reviewing again :-D

Reply:
Done.